### ISSUE TITLE
Failed to convert onnx model with LSTM layer

### ISSUE THREAD

**Issue by aabghari** [alibaba/MNN#1890]
# 平台(如果交叉编译请再附上交叉编译目标平台):  
# Platform(Include target platform as well if cross-compiling):  
Linux debian-10 64bit

# Github版本:
# Github Version:
``commit 344629419f710b39559e616f1af7bb7bd29617db (HEAD -> master, tag: 1.2.7, origin/master, origin/HEAD, v1.2.7)``

# 编译方式:
# Compiling Method

```
cmake -DMNN_USE_SYSTEM_LIB=OFF -DMNN_BUILD_SHARED_LIBS=OFF -DMNN_SEP_BUILD=OFF -DMNN_BUILD_DEMO=OFF -DMNN_BUILD_TOOLS=ON -DMNN_BUILD_CONVERTER=ON -DMNN_EVALUATION=OFF -DMNN_OPENCL=OFF -DMNN_OPENGL=OFF -DMNN_VULKAN=OFF -DMNN_AVX512=ON -DMNN_ARM82=OFF -DMNN_SUPPORT_BF16=OFF ../
```
Converting the attached onnx model to MNN fails with the following error, however the same model gets converted successfully with version 1.2.6.

```
$ MNNConvert -f ONNX --modelFile model.onnx --MNNModel model.mnn --bizCode MNN 
Start to Convert Other Model Format To MNN Model...
[18:26:09] MNN/tools/converter/source/onnx/onnxConverter.cpp:30: ONNX Model ir version: 7
[18:26:09] MNN/tools/converter/source/onnx/onnxConverter.cpp:98: Check it out ==> 63 has empty input, the index is 3
[18:26:09] MNN/tools/converter/source/onnx/onnxConverter.cpp:98: Check it out ==> 63 has empty input, the index is 4
Start to Optimize the MNN Net...
Broad cast error, dim1 = 16, dim2 = 24
MNN LSTM not support optional 4th input (must provide B)
Convert Onnx's Op 63 , type = LSTM, failed, may be some node is not const
MNN LSTM not support optional 4th input (must provide B)
Convert Onnx's Op 63 , type = LSTM, failed, may be some node is not const
Broad cast error, dim1 = 16, dim2 = 24
[18:26:09] MNN/tools/converter/source/common/writeFb.cpp:82: These Op Not Support: ONNX::LSTM 
Converted Failed!

```

[model.onnx.zip](https://github.com/alibaba/MNN/files/8328282/model.onnx.zip)

**Comment by jxt1234** [comment]
We will check it.

**Comment by jxt1234** [comment]
Fixed by  951a57b07c34acd31f3687d6225c674d0c53a211
